lib/elixir/lib/code.ex

```diff
@@ -206,6 +206,240 @@ defmodule Code do
     required_files()
   end
 
+  @doc """
+  Receives a string and returns an autocompletion context.
+
+  This function provides a best-effort for autocompletion which
```

Comment:
This function name violates the principle of least surprise. IMO change from `autocomplete` to `autocompletion` did little to resolve it. Also from the docs it's not clear how and why would I want to use it.

Reply:
"principle of least surprise" is not a good argument. Each person has their own expectations and different things to surprise them. Can you please provide more concrete feedback?

Reply:
In any case, I was already planning a rename to cursor_context, so please check the update docs. :)

Reply:
True, I do have specific expectations but autocomplete is pretty well understood term even outside IT bubble (https://en.wikipedia.org/wiki/Autocomplete)